pkg/tbtc/tbtc.go

```diff
@@ -339,6 +339,7 @@ func Initialize(
 
 				node.handleDepositSweepProposal(
 					event.Proposal,
+					expiresAt,
```

Comment:
Really nice approach to determine it from the `WalletCoordinator`'s lock and then subtract `dsa.signingTimeoutSafetyMargin`! 🙌🏻